web/lib/api/metrics/costCalc.ts

```diff
@@ -19,7 +19,7 @@ const COSTS_PROMPT = {
   "gpt-3.5-turbo-0613": 0.0015,
   "gpt-35-turbo-16k": 0.003,
   "gpt-3.5-turbo-16k-0613": 0.003,
-  "text-embedding-ada-002-v2": 0.0001,
```

Comment:
Let's make it so that we can have both for backwards compatibility

```
"text-embedding-ada-002": 0.0001,
"text-embedding-ada-002-v2": 0.0001,
```